network/datadog_checks/network/network.py

```diff
@@ -438,7 +443,9 @@ def _parse_linux_cx_state(self, lines, tcp_states, state_col, protocol=None, ip_
         Parse the output of the command that retrieves the connection state (either `ss` or `netstat`)
         Returns a dict metric_name -> value
         """
-        metrics = dict.fromkeys(self.cx_state_gauge.values(), 0)
+        metrics = {}
+        for _, val in iteritems(self.cx_state_gauge.values):
```

Comment:
I think we can remove the call to `values` here

Reply:
oh boop thanks